core/tests/protractor_desktop/adminTabFeatures.js

```diff
@@ -51,6 +53,12 @@ describe('Admin Page', function() {
     await adminPage.updateRole('collectionEditor1', 'collection editor');
     await adminPage.getUsersAsssignedToRole('collection editor');
     await adminPage.expectUsernamesToMatch(['collectionEditor1']);
+
+    await adminPage.get();
+    await adminPage.addReviewer('newQuestionReviewer', 'QUESTION');
+    await adminPage.getReviewersAssignedToRole('question');
```

Comment:
Why does the capitalization of the role change?

Reply:
I think `action.sendKeys` is fine. I'm wondering though why for `addReviewer` the role is `QUESTION` but in `getReviewersAssignedToRole` the role is `question`. Shouldn't the capitalization be the same between the two lines?

Reply:
Yes - it was capitalized for when I was using cssContainingText because it's capitalized from a user's perspective on the site (if you look at the dropdown for reviewer roles they're all capitalized for whatever reason). I changed QUESTION to lower case on line 58 after I removed the cssContainingText and switched it to sendKeys - I must've changed it back again by accident - my bad :)